columnflow/tasks/plotting.py

```diff
@@ -220,14 +209,20 @@ class PlotVariables1D(
     PlotVariablesBaseSingleShift,
     PlotBase1D,
 ):
-    pass
+    plot_function = PlotBase.plot_function.copy(
+        default="columnflow.plotting.example.plot_variable_per_process",
+        add_default_to_description=True,
```

Comment:
where is the implementation of the `add_default_to_description` functionality?

Reply:
It's in law at https://github.com/riga/law/blob/02dc3744f87fa0a6c2202ff0dc8b80ca885f443b/law/patches.py#L369-L399

I'm going to open a PR directly to luigi, but in the meantime we can use that "patch".